src/nixos-anywhere.sh

```diff
@@ -11,6 +11,10 @@ Options:
   set the flake to install the system from.
 * -i <identity_file>
   selects which SSH private key file to use.
+* -p, --ssh-port <ssh_port>
+  set the ssh port to connect with
+* -o, --ssh-option <ssh_option>
```

Comment:
I think I want to reserve -o for something else, it's not super descriptive on its own.

```suggestion
* --ssh-option <ssh_option>
```